scripts/create-k3s-cluster-with-SSL-passthrough.sh

```diff
@@ -103,11 +103,25 @@ helm upgrade --install ingress-nginx ingress-nginx \
 
 if [[ "$wait" == "true" ]] ; then
   echo "Waiting for nginx ingress with SSL passthrough to complete..."
+  max_attempts=60  # 5 minutes if sleep=5
+  attempt=0
   while [ -z "$(kubectl get pod --namespace ingress-nginx --selector=app.kubernetes.io/component=controller --no-headers -o name 2> /dev/null)" ] ;  do
       sleep 5
+      attempt=$((attempt+1))
+      if [ $attempt -ge $max_attempts ]; then
+        echo "ERROR: Timed out waiting for nginx ingress controller pod to appear."
+        kubectl get pods -n ingress-nginx -o wide || true
+        exit 1
```

Comment:
We learned at KubeCon that sometimes people try our stuff in an astonishingly slow network environment. We should be removing or lengthening timeouts, not adding them or making them shorter.